lib/fog/vsphere/requests/compute/list_clusters.rb

```diff
@@ -27,41 +33,41 @@ def get_raw_clusters_from_folder(folder)
           end.flatten
         end
 
-        def cluster_attributes cluster, datacenter_name
+        def cluster_attributes(cluster, datacenter_name)
           {
-            :id             => managed_obj_id(cluster),
-            :name           => cluster.name,
-            :full_path      => cluster_path(cluster, datacenter_name),
-            :num_host       => cluster.summary.numHosts,
-            :num_cpu_cores  => cluster.summary.numCpuCores,
-            :overall_status => cluster.summary.overallStatus,
-            :datacenter     => datacenter_name || parent_attribute(cluster.path, :datacenter)[1],
+            id: managed_obj_id(cluster),
```

Comment:
At this time, fog-vsphere still supports Ruby 1.8.7 (I know, I know, I'll make that go away soon and start only supporting 2.3+) so this Hash syntax needs to be changed back.